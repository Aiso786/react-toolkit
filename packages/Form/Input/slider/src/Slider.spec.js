import React from 'react';
import { shallow } from 'enzyme';
import Slider from './Slider';

describe('<Slider>', () => {
  it('renders Slider correctly', () => {
    const wrapper = shallow(
      <Slider label="File *" id="id" name="file" onChange={() => {}} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
