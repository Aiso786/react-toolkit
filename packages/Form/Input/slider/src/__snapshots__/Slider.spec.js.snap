// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slider> renders Slider correctly 1`] = `
<div
  style={
    Object {
      "margin": 0,
      "width": "100%",
    }
  }
>
  <ComponentEnhancer(undefined)
    activeDotStyle={Object {}}
    className=""
    defaultValue={1}
    disabled={false}
    dotStyle={Object {}}
    dots={false}
    handle={[Function]}
    handleStyle={
      Array [
        Object {},
      ]
    }
    included={true}
    marks={
      Object {
        "0": "VERY BAD",
        "1": "BAD",
        "2": "NICE",
        "3": "VERY NICE",
      }
    }
    max={3}
    min={0}
    onAfterChange={null}
    onBeforeChange={null}
    onChange={[Function]}
    prefixCls="rc-slider"
    railStyle={Object {}}
    readOnly={false}
    step={null}
    trackStyle={
      Array [
        Object {},
      ]
    }
    vertical={false}
  />
</div>
`;
